CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+This is the new, unified, changelog that contains changes from across all libcnb.rs crates. Before version `0.10.0`,
+separate changelogs for each crate were used. If you need to refer to these old changelogs, find them named
+`HISTORICAL_CHANGELOG.md` in their respective crate directories.
+
+## [Unreleased]
+
+Highlight of this release is the bump to
+[Buildpack API 0.8](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.8) which brings support for SBOM to
+libcnb.rs. This is also the first release where all libcnb.rs crates are released at the same time and with the same
+version number. See the changelog below for other changes.
+
+### Changes
```

Comment:
```suggestion
### Changed
```

(To make tense consistent with the other sections, plus the suggested categories from https://keepachangelog.com/en/1.0.0/ )